ArduPlane/mode_auto.cpp

```diff
@@ -159,3 +159,25 @@ bool ModeAuto::_pre_arm_checks(size_t buflen, char *buffer) const
     // Note that this bypasses the base class checks
     return true;
 }
+
+bool ModeAuto::is_landing() const
+{
+    switch(plane.mission.get_current_nav_cmd().id) {
+    case MAV_CMD_NAV_LAND:
+    case MAV_CMD_NAV_VTOL_LAND:
+    case MAV_CMD_DO_VTOL_TRANSITION:
```

Comment:
VTOL transition is not a landing.

Reply:
removed the whole switch and using flight stage